src/libcore/result.rs

```diff
@@ -1189,12 +1190,15 @@ impl<T: DerefMut, E> Result<T, E> {
     /// # Examples
     ///
     /// ```
+    /// #![feature(inner_deref)]
+    /// let mut s = "HELLO".to_string();
     /// let mut x: Result<String, u32> = Ok("hello".to_string());
```

Comment:
Was wondering why CI doesn't catch this?

Reply:
CI failed to run doctests at all. This PR fixes that. See https://github.com/rust-lang/rust/issues/73286.

Reply:
Thanks. I should had read the link issue.